frontend/src/scenes/insights/Insight.tsx

```diff
@@ -86,11 +86,6 @@ export function Insight({ shortId }: { shortId?: InsightShortId } = {}): JSX.Ele
         l: {
             action: () => handleHotkeyNavigation(InsightType.LIFECYCLE, 'l'),
         },
-        escape: {
-            // Exit edit mode with Esc. Full screen mode is also exited with Esc, but this behavior is native to the browser.
-            action: () => setInsightMode(ItemMode.View, InsightEventSource.Hotkey),
-            disabled: insightMode !== ItemMode.Edit,
-        },
         e: {
             action: () => setInsightMode(ItemMode.Edit, InsightEventSource.Hotkey),
```

Comment:
```suggestion
            action: () => setInsightMode(insightMode === ItemMode.Edit ? ItemMode.View : ItemMode.Edit, InsightEventSource.Hotkey),
```

Reply:
I tried this out, but it's quite a confusing action to exit edit mode having neither discarded changes nor saved them – the results in view mode don't match up with what's saved then. For now I'll opt for the initial solution of just disabling Escape.

Reply:
Oh you're right that makes sense, thanks @Twixes